rs/verifier/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "verity-verifier"
```

Comment:
Naming conventions here will change - ie. `verity-verify-tls`
Whereby `verity` is workspace, `verify` is the purpose, `tls` is the suffix